### PR TITLE
Finished Amber "Drain Her"

### PR DESCRIPTION
**Pull request by legojohn**
Intro, Throat and Ass are done.
Pussy is missing content in source material.

Button disabled until finished.

### CODE REVIEW

**Comment by legojohn** [comment]
I don't know why this shows the henderson.png as removed, my git client seems to be having some kind of lower-/uppercase issue with the hr / HR parent folder.

**Comment by legojohn** [comment]
Ready to call this finished, will playtest some more just in case, but so far it's looking good!

File: includes/follower/amber.as [hunk 1]
```diff
@@ -2675,10 +2672,242 @@ public function amberOnShipSexDrainHer():void
...
+		output("\n\nSettling into the rhythm, you pull back gently in time with Amber and push forward when she re-hilts herself deep inside you to deposit yet another hot, heavy load of her sperm, enjoying the way she throbs and groans in outright bliss. You're giving her the time of her life, and there's going to be a <i>lot</i> more where that's coming from before the two of you are done. Closing your eyes, you hollow your cheeks and suck on her big, pulsating cock as hard as you can.");
+		output("\n\n“Ho-o-ohhh, god,” Amber groans, squeezing her hindquarters and letting out another heated blossom of seed into your insides. “That- that was amazing...”");
+	}
+	
+	output("\n\nYou grin around her enormous cock. Still stiff and needy, she's obviously up to blow several more loads inside you. But where?");
+	
+	//Sex is not *actually* a time machine, so let's pass some time
+	processTime(10 + rand(10));
+
```
Comment (Fenoxo) [inline includes/follower/amber.as]: Probably needs lust raised.

Comment (legojohn) [reply includes/follower/amber.as]: There already is, seems you reviewed one of the earlier commits. Or I fucked up majorly in using git :/

Comment (Fenoxo) [reply includes/follower/amber.as]: Ah no, that's on my end for paging through these one at time.

Comment (Fenoxo) [reply includes/follower/amber.as]: Suppose I should just check out the whole branch and review the final code.

Comment (legojohn) [reply includes/follower/amber.as]: You can just go to changed files and see what I did vs. original. https://github.com/OXOIndustries/TiTS-Public/pull/205/files

**Comment by Fenoxo** [comment]
Hey I checked it out and did some reviewing. Pinged you with it all on discord. Lemme know if you need anything from me to help clean up the few rough patches!

**Comment by Fenoxo** [comment]
Yeah all that shit you did with renaming everything multiple times is liable to cause issues, so now I'm gonna have to pull out the code by hand. Please don't do that again.

**Pull request closed without merging**